components/SplitwiseSummary/SplitwiseSummary.js

```diff
@@ -11,30 +12,47 @@ function SplitwiseSummary() {
     debtsSummary: { toPay, toCollect },
   } = useSplitwiseSummary();
 
+  const [fontsLoaded] = useFonts({
+    SpaceMonoItalic: require('../../assets/fonts/SpaceMono-BoldItalic.ttf'),
+    SpaceMonoRegular: require('../../assets/fonts/SpaceMono-Regular.ttf'),
+  });
+
+  if (!fontsLoaded) {
+    return <AppLoading />;
+  }
+
   return (
     <View style={styles.componentContainer}>
-      <Text style={styles.titleText}>Resumen Splitwise</Text>
+      <Text
+        style={{ ...styles.titleText, ...{ fontFamily: 'SpaceMonoRegular' } }}
+      >
+        Resumen Splitwise
+      </Text>
       <View style={styles.rowContainer}>
-        <View>
-          <Text style={styles.toCollectTitle}>x Cobrar</Text>
-          {Object.keys(toCollect)
+        <View style={styles.less}>
+          {Object.keys(toPay)
             .sort()
             .map(currency => (
-              <Text key={currency} style={styles.coinText}>{` ${formatCurrency(
-                toCollect[currency],
-                currency
-              )}`}</Text>
+              <Text
+                key={currency}
+                style={{
+                  ...styles.coinText,
+                  ...{ fontFamily: 'SpaceMonoRegular' },
```

Comment:
Lo otro, si quieres definir en particular una propiedad del style, no es necesario usar `...`, basta con hacer:
```suggestion
                  fontFamily: 'SpaceMonoRegular',
```